src/test/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilderTest.java

```diff
@@ -26,60 +26,76 @@ void buildSearchConceptsQuery(CloudPlatform platform) {
-                + "AND (CONTAINS_SUBSTR(c.concept_name, 'cancer') "
-                + "OR CONTAINS_SUBSTR(c.concept_code, 'cancer')))",
-            platformWrapper);
-    assertThat(
-        "generated SQL for search concepts query is correct",
-        actual,
-        equalToCompressingWhiteSpace(expected));
+    if (platformWrapper.isGcp()) {
+      assertThat(
+          "generated SQL for GCP is correct",
+          actual,
+          equalToCompressingWhiteSpace(
+              "SELECT c.concept_name, c.concept_id FROM concept AS c "
+                  + "WHERE (c.domain_id = 'condition' "
+                  + "AND (CONTAINS_SUBSTR(c.concept_name, 'cancer') "
+                  + "OR CONTAINS_SUBSTR(c.concept_code, 'cancer'))) "
+                  + "LIMIT 100"));
+    }
+    if (platformWrapper.isAzure()) {
+      assertThat(
+          "generated SQL for Azure is correct",
+          actual,
+          equalToCompressingWhiteSpace(
+              "SELECT TOP 100 c.concept_name, c.concept_id FROM concept AS c "
+                  + "WHERE (c.domain_id = 'condition' "
+                  + "AND (CHARINDEX('cancer', c.concept_name) > 0 "
+                  + "OR CHARINDEX('cancer', c.concept_code) > 0))"));
+    }
   }
 
   @ParameterizedTest
   @EnumSource(CloudPlatform.class)
   void buildSearchConceptsQueryEmpty(CloudPlatform platform) {
-    CloudPlatformWrapper cloudPlatformWrapper = CloudPlatformWrapper.of(platform);
+    CloudPlatformWrapper platformWrapper = CloudPlatformWrapper.of(platform);
     String actual =
         SearchConceptsQueryBuilder.buildSearchConceptsQuery(
             "Condition", "", s -> s, CloudPlatformWrapper.of(platform));
     String expected =
-        formatSQLWithLimit(
-            "SELECT c.concept_name, c.concept_id FROM concept AS c WHERE c.domain_id = 'Condition'",
-            cloudPlatformWrapper);
-    assertThat(
-        "generated SQL for empty search concepts query is correct",
-        actual,
-        equalToCompressingWhiteSpace(expected));
-  }
-
-  String formatSQLWithLimit(String sql, CloudPlatformWrapper cloudPlatformWrapper) {
-    int limit = 100;
-    if (cloudPlatformWrapper.isAzure()) {
-      return String.format("TOP %d %s", limit, sql);
-    } else if (cloudPlatformWrapper.isGcp()) {
-      return String.format("%s LIMIT %d", sql, limit);
-    } else {
-      throw new NotImplementedException("Cloud platform not implemented");
+        "c.concept_name, c.concept_id FROM concept AS c " + "WHERE c.domain_id = 'Condition' ";
+    if (platformWrapper.isAzure()) {
+      assertThat(
+          "generated SQL for Azure empty search string is correct",
+          actual,
+          equalToCompressingWhiteSpace("SELECT TOP 100 " + expected));
+    }
+    if (platformWrapper.isGcp()) {
+      assertThat(
+          "generated SQL for GCP empty search string is correct",
+          actual,
+          equalToCompressingWhiteSpace("SELECT " + expected + "LIMIT 100"));
```

Comment:
Nit: Even if it passes because it removes the whitespace, do you think it would make sense to add a space before `LIMIT 100`? Or remove the space after `SELECT `?